src/System.Private.Interop/src/System/Runtime/InteropServices/ComTypes/typedesc.cs

```diff
@@ -9,7 +9,8 @@ namespace System.Runtime.InteropServices.ComTypes
     [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
     public struct TYPEDESC
     {
-        public IntPtr lpValue;
+		[System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA2111:PointersShouldNotBeVisible")]
```

Comment:
Nit: these edits are all using tabs instead of spaces. CoreFX guidelines (and the rest of the file) use spaces.

Reply:
Yes. Fixed these.